typespec-extension/package.json

```diff
@@ -70,5 +70,9 @@
     "@typescript-eslint/parser": "~5.48.0",
     "@typescript-eslint/eslint-plugin": "~5.48.0",
     "eslint-plugin-deprecation": "~1.3.3"
+  },
+  "overrides": {
```

Comment:
What is the override here used for?  To resolve `pnpm` and `npm` error?

Reply:
To fix the resolution error like this https://dev.azure.com/azure-sdk/internal/_build/results?buildId=2751024&view=logs&j=ca395085-040a-526b-2ce8-bdc85f692774&t=cf4477f9-109f-5399-1612-bcbe5909c9f2 (from tsp packages on their peer dependency)

Reply:
Emm..The error is due to version conflict, from my understanding it is because typespec-java depends on typespec-azure dev versioin and typespec dev version, while typespec-azure dev version depends on typespec stable version, so **one depends on typespec stable version, the other one depends on typespec dev version**, so it has version conflict. Don't understand why `overrides` could solve the issue..

Reply:
I think `overrides` just means we use that version of that package, whatever other dependencies says.